vue/components/ui/molecules/filter/filter.vue

```diff
@@ -142,6 +143,10 @@ export const Filter = {
             type: String,
             default: null
         },
+        rowSelection: {

```

Comment:
Just wanted to say that "selectableRows" would be a better name, similar to `clickableRows`... but I noticed `table` uses "rowSelection" prop name for quite some time and would be a breaking change :disappointed: